recipe/meta.yaml

```diff
@@ -23,24 +23,22 @@ requirements:
   build:
     - {{ compiler('cxx') }}
   host:
-    - python
+    - python !=3.11.0,!=3.11.1
     - pip
-    - setuptools
+    - setuptools >=61.2
     - wheel
-    - setuptools_scm
-    - toml
+    - setuptools_scm >=3.4.3
     - cppy 1.2.1
-    - bytecode 0.14.0
   run:
     - python
-    - python.app  # [osx]
+    # - python.app  # [osx]
```

Comment:
Is this still needed?

Reply:
I believe it is still needed. Because of `osx_is_app: true`. 
FYI the python.app recipes says _Proxy on macOS letting Python libraries hook into the GUI event loop_
https://github.com/AnacondaRecipes/python.app-feedstock/blob/master/recipe/meta.yaml